shared/cluster.go

```diff
@@ -277,6 +277,16 @@ func SonobuoyMixedOS(action, version string) error {
 	return err
 }
 
+// PrintClusterState prints the output of kubectl get nodes,pods -A -o wide
+func PrintClusterState() {
+	cmd := "kubectl get nodes,pods -A -o wide --kubeconfig=" + KubeConfigFile
+	res, err := RunCommandHost(cmd)
+	if err != nil {
+		_ = ReturnLogError("failed to print cluster state: %w\n", err)
+	}
+	fmt.Println("\n", res)
```

Comment:
could you add something like:

Cluster state right before upgrade:
....


something like that